### PR TITLE
APP-4097 - Toast component

### PR DESCRIPTION
**Pull request by axeleriksson147**
- https://perzoinc.atlassian.net/wiki/spaces/DevX/pages/1930300829/Toast - Proposal
- https://perzoinc.atlassian.net/browse/APP-4097 - Ticket
- https://www.figma.com/file/V0VKw74594EyCKHwqYk6LV/%E2%9D%96-SDS-Component-Library - Design for Toast component
- ui-toolkit-styles PR - https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-styles/pull/87

Screenshot 2021-04-30 at 12 13 20
Screenshot 2021-05-04 at 13 02 06

### CODE REVIEW

**Comment by jakub-symphony** [comment]
@axeleriksson147 - could you check if the corner radius = 8px ? It seems it's 16px or so...

**Comment by axeleriksson147** [comment]
@jakub-symphony fixed, not sure why I put 16. Thanks!

**Review by symphony-jean-michael — APPROVED** [review]
LGTM. Thank you @axeleriksson147 for your contribution! 👍  :)

File: src/components/toast/Toast.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+interface Placement {
+    horizontal: 'left' | 'center' | 'right';
+    vertical: 'top' | 'center' | 'bottom';
+}
+
+export interface ToastProps extends Omit<React.HTMLProps<HTMLDivElement>, 'children' | 'content'> {
+    /** Optional CSS class name */
+    className?: string;
+    /** If true, close icon will be placed to the right */
+    closeIcon?: boolean;
```
Comment (AliciaSymphony) [inline src/components/toast/Toast.tsx:16]: by curiosity, how the Toast closes he we don't have the close icon? (timeout?)

File: stories/Toast.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        JSON.stringify({ horizontal: 'center', vertical: 'bottom'}),
+        JSON.stringify({ horizontal: 'right', vertical: 'bottom'})
+      ],
+    },
+  },
+};
+
+const Template = (args) => {
+  const { placement, ...restArgs } = args;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (AliciaSymphony) [inline stories/Toast.stories.tsx:35]: is this error still appearing? I see that you are using them

**Review by AliciaSymphony — APPROVED** [review]
Great work! Thank you for your contribution Axel

File: stories/Toast.stories.tsx (hunk 2 above)
Comment (axeleriksson147) [reply stories/Toast.stories.tsx:35]: Thanks, will remove.

File: src/components/toast/Toast.tsx (hunk 1 above)
Comment (axeleriksson147) [reply src/components/toast/Toast.tsx:16]: So show props toggles visibility, onClickClose you should pass in a function that as a "suggestion" sets show to false.

I think future development of Toast will have a timeout, can be the same base logic as for Tooltip. Then we'll probably keep the show state inside the component.

**Comment by axeleriksson147** [comment]
Need to update with new styles version.

**Pull request merged**